test/functional/testlib.bash

```diff
@@ -1188,6 +1201,42 @@ update_manifest() { # swupd_function
 
 }
 
+get_entry_from_manifest() { # swupd_function
```

Comment:
I liked this function too and I can see several usages for that. But there are some problems